pkg/command/ceip_participation.go

```diff
@@ -53,10 +53,10 @@ func newCEIPParticipationSetCmd() *cobra.Command {
 		Long:  "Set the opt-in preference for CEIP (subject to change)",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !strings.EqualFold(args[0], "true") && !strings.EqualFold(args[0], "false") {
+			if !strings.EqualFold(args[0], True) && !strings.EqualFold(args[0], "false") {
```

Comment:
Can you explain what the point of the change is? How was it causing problems, and why is handling of false and true different?

Reply:
Added True and False const

Reply:
Updated to use the strings.EqualFold()